src/rules/pylint/rules/bidirectional_unicode.rs

```diff
@@ -0,0 +1,49 @@
+use crate::ast::types::Range;
+use crate::checkers::ast::Checker;
+use crate::define_violation;
+use crate::registry::Diagnostic;
+use crate::violation::Violation;
+
+use ruff_macros::derive_message_formats;
+use rustpython_ast::Expr;
+
+const BIDI_UNICODE: [char; 10] = [
+    '\u{202A}', //{LEFT-TO-RIGHT EMBEDDING}
+    '\u{202B}', //{RIGHT-TO-LEFT EMBEDDING}
+    '\u{202C}', //{POP DIRECTIONAL FORMATTING}
+    '\u{202D}', //{LEFT-TO-RIGHT OVERRIDE}
+    '\u{202E}', //{RIGHT-TO-LEFT OVERRIDE}
+    '\u{2066}', //{LEFT-TO-RIGHT ISOLATE}
+    '\u{2067}', //{RIGHT-TO-LEFT ISOLATE}
+    '\u{2068}', //{FIRST STRONG ISOLATE}
+    '\u{2069}', //{POP DIRECTIONAL ISOLATE}
+    // The following was part of PEP 672:
+    // https://www.python.org/dev/peps/pep-0672/
+    // so the list above might not be complete
+    '\u{200F}', //{RIGHT-TO-LEFT MARK}
+                // We don't use
+                //   "\u200E" # \n{LEFT-TO-RIGHT MARK}
+                // as this is the default for latin files and can't be used
+                // to hide code
+];
```

Comment:
What's the source for this list?

Reply:
https://github.com/PyCQA/pylint/blob/main/pylint/checkers/unicode.py#L37

Reply:
Thank you :)